From 06a160189d0f840bb81e76bff29766305a7d5162 Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu00@gmail.com>
Date: Wed, 6 Jul 2011 22:11:32 -0700
Subject: [PATCH 20/65] FLUME-701: Make flume project tar generated contents live inside ./flume-<version>/

Currently, when a tarball is generated by the flume-distribution module, all
contents within the tarball live in ./flume-distribution-<version>/. This patch
changes it so that contents within the tarball lives in ./flume-<version>/
---
 flume-distribution/src/main/assembly/dist-bin.xml |    2 ++
 flume-distribution/src/main/assembly/dist.xml     |    2 ++
 2 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/flume-distribution/src/main/assembly/dist-bin.xml b/flume-distribution/src/main/assembly/dist-bin.xml
index 0deb1c0..e153204 100644
--- a/flume-distribution/src/main/assembly/dist-bin.xml
+++ b/flume-distribution/src/main/assembly/dist-bin.xml
@@ -12,6 +12,8 @@
     <format>tar.gz</format>
     <format>zip</format>
   </formats>
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <baseDirectory>flume-${project.version}</baseDirectory>
 
   <moduleSets>
 
diff --git a/flume-distribution/src/main/assembly/dist.xml b/flume-distribution/src/main/assembly/dist.xml
index adecb35..28474dc 100644
--- a/flume-distribution/src/main/assembly/dist.xml
+++ b/flume-distribution/src/main/assembly/dist.xml
@@ -12,6 +12,8 @@
     <format>tar.gz</format>
     <format>zip</format>
   </formats>
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <baseDirectory>flume-${project.version}</baseDirectory>
 
   <moduleSets>
 
-- 
1.7.0.4

